src/Storage/Storage.Autorest/custom/Start-AzStorageAccountMigration.ps1

```diff
@@ -199,11 +198,21 @@ param(
     [Microsoft.Azure.PowerShell.Cmdlets.Storage.Category('Runtime')]
     [System.Management.Automation.SwitchParameter]
     # Use the default credentials for the proxy
-    ${ProxyUseDefaultCredentials}
+    ${ProxyUseDefaultCredentials},
+    
+    [Parameter()]
+    # Forces the cmdlet to convert the account's redundancy configuration without prompting for confirmation.
+    [System.Management.Automation.SwitchParameter]
+    ${Force}
 )
 
     process {
-        Write-Warning("After your request to convert the account's redundancy configuration is validated, the conversion will typically complete in a few days, but can take a few weeks depending on current resource demands in the region, account size, and other factors. The conversion can't be stopped after being initiated, and for accounts with geo redundancy a failover can't be initiated while conversion is in progress. The data within the storage account will continue to be accessible with no loss of durability or availability.")
+      if ($PSBoundParameters.ContainsKey("Force")) {
+        $PSBoundParameters.Remove("Force")  | Out-Null
+      }
+      
+      if ($Force.IsPresent -or $PsCmdlet.ShouldContinue("", "After your request to convert the account's redundancy configuration is validated, the conversion will typically complete in a few days, but can take a few weeks depending on current resource demands in the region, account size, and other factors. The conversion can't be stopped after being initiated, and for accounts with geo redundancy a failover can't be initiated while conversion is in progress. The data within the storage account will continue to be accessible with no loss of durability or availability.")) {
```

Comment:
The sentence should be : "After your request to convert the account’s redundancy configuration is validated, the conversion will typically complete in a few days, but can take several weeks depending on current resource demands in the region, account size, and other factors. The conversion can’t be stopped after being initiated, and for accounts with geo redundancy a failover can’t be initiated while conversion is in progress. The data within the storage account will continue to be accessible with no loss of durability or availability. Confirm redundancy configuration change:"

Please make sure it display correctly in both PSH5 and PSH7.